### ISSUE TITLE
Can this component support TCP protocol

### ISSUE THREAD

**Issue by shangjunshang** [spring-cloud/spring-cloud-openfeign#662]
Can this component support the TCP protocol? If so, what can I do? Thank you for your answer

**Comment by ciscoo** [comment]
Someone correct me if I'm wrong here, but no. Spring Cloud OpenFeign configures Feign clients which are HTTP clients. 

If you need TCP transport, then have a look at RSocket https://rsocket.io/

**Comment by shangjunshang** [comment]
Thank you for your answer. Are you the author of this component

**Comment by OlgaMaciaszek** [comment]
Yes, Spring Cloud OpenFeign project provides Spring integration for OpenFeign that, in turn, uses HTTP clients under the hood.
